Client/src/Pages/Monitors/Home/actionsMenu.jsx

```diff
@@ -155,6 +180,20 @@ const ActionsMenu = ({ monitor, isAdmin, updateCallback }) => {
             Clone
           </MenuItem>
         )}
+        {isAdmin && (
+          <MenuItem
+            onClick={(e) => {
+              e.stopPropagation();
+              handlePause(e);
+            }}
+          >
+          {monitor?.isActive ? (
+            "Pause"
+          ) : (
+            "Resume"
+          )}
```

Comment:
The regular braces are not needed here:

`{monitor?.isActive  === true ? "Pause" : "Resume"}`

is sufficient :+1: